scratch/app-github.jsonc

```diff
@@ -0,0 +1,39 @@
+{
+    "type": "view",
+    "bindings": [
+        {
+            "type": "menu",
+            "bindings": [
+                // Are these menu items meant to lead to rendering the list of open PRs with Mattermost, or is it a link to a page on GitHub with these listed?
+                {
+                    "type": "menu_item",
+                    "label": "Your Pull Requests",
+                    "icon": "icons/code-merge.png",
+                    "hint": "1",
+                    "submit": "views/your-pull-requests"
+                },
+                {
+                    "type": "menu_item",
+                    "label": "Pull Requests Needing Review",
+                    "icon": "icons/eye.png",
+                    "hint": "1",
+                    "submit": "views/your-review-requests"
```

Comment:
0/5 but mostly a question for design. The arrow implies something will be shown here, right? If we had no arrow, we would expect an action to happen or a new page to be open (at least that is my impression). If that is true, should we add some field to either:
- decide the right icon
- decide whether to put an arrow or not
- select from a set of icons (left arrow, down arrow, nothing...)

Reply:
If you're referring to the arrow on the right side, it's an indicator that there is another page to view all your PRs

![image](https://user-images.githubusercontent.com/2040554/186915347-a52f0258-9cc9-46a7-9d80-7145099cbd6e.png)

Reply:
Yes, that is exactly the arrow I mean. The thing is... with the way the model is done right now, there is no way to decide whether to show or not the arrow. Do we envision any menu like this that not every element opens a new view?

Reply:
@matthewbirtch @larkox

> The thing is... with the way the model is done right now, there is no way to decide whether to show or not the arrow. Do we envision any menu like this that not every element opens a new view?

In the Compass Component designs for menus, there are many variations laid out https://zeroheight.com/29be2c109/p/1092d5-menu-item. The question is, how do these all relate to the functionality we want to support in the App Framework? We won't be able to express all of the different variations with JSON data. Which variations do we _need_ to support for the views we plan to build for Apps? What should we support for building more views in the future that aren't designed yet?